filemage/datadog_checks/filemage/__about__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
```suggestion
__version__ = '1.0.0'
```

Reply:
resolved in 743314bd136fa41a564d99d590471733f4969a6b